render.js

```diff
@@ -0,0 +1,46 @@
+const Renderer = function(){
+    let updatedPosts=tweeter.getPosts()
+  
+       $("#posts").empty()
+ 
+       for(let post of updatedPosts){
+    let updatedComment=` `
+    for(const comment of post.comments){
+        updatedComment+=`
+        <div class="comment" id='${comment.id}'>
+        <div class="delete-comment">X</div>
+        <p class="comment-text">${comment.text}</p>
+ 
+    </div>
+          
+        
+        ` 
+    }
+ 
+ let updatedPosts= $(`     <div class="post"  id="${post.id}">
```

Comment:
you want to store js information in `data-id` attribute. the id attribute is for css